### PR TITLE
Add support for Ubuntu bionic

### PR DESCRIPTION
**Pull request by daniel-zullo**
- Remove D1 flags which are no longer needed as the code base is only D2.
- Set libssl and libcrypto to version 1.0.0 supported in Ubuntu xenial and bionic.
- The dmd-transitional compiler is available for Ubuntu bionic only in vesion 2.078.3.s*.
- Update beaver build image to support Ubuntu xenial and bionic distributions.
- Remove package d1to2fix that is no longer required as the code base is only D2.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -47,6 +47,17 @@ jobs:
           env: DMD=2.085.* F=production
         - <<: *test-matrix-d2
           env: DMD=2.085.* F=devel
+        - <<: *test-matrix-d2
+          env: DIST=bionic DMD=2.078.3.s* F=production
```
Comment (mihails-strasuns) [inline .travis.yml:51]: Adding explicit `DIST=xenial` to old rows would make it slightly clearer without any further context.

**Review by mihails-strasuns — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (daniel-zullo) [reply .travis.yml:51]: right, I will add it to the old rows soon

**Comment by daniel-zullo** [comment]
This is now blocked due to the issue that Don reported in 
https://issues.dlang.org/show_bug.cgi?id=20441

**Comment by Geod24** [comment]
That bug has been fixed

**Comment by daniel-zullo** [comment]
Updated to test dmd 2.091

**Comment by daniel-zullo** [comment]
@Geod24 you might want to review it.

**Review by Geod24 — APPROVED** [review]
LGTM

**Pull request merged**